README.md

```diff
@@ -1,18 +1,20 @@
-# miniforge
-![Build miniforge](https://github.com/conda-forge/miniforge/workflows/Build%20miniforge/badge.svg)
+# Miniforge
+![Build Miniforge](https://github.com/conda-forge/miniforge/workflows/Build%20miniforge/badge.svg)
 
-This repository holds a minimal installer for conda specific to conda-forge. It is comparable to [Miniconda](https://docs.conda.io/en/latest/miniconda.html), but with
+This repository holds a minimal installer for [Conda](https://conda.io/) specific to [Conda Forge](https://conda-forge.org/). It is comparable to [Miniconda](https://docs.conda.io/en/latest/miniconda.html), but with:
```

Comment:
```suggestion
This repository holds a minimal installer for [Conda](https://conda.io/) specific to [conda-forge](https://conda-forge.org/). It is comparable to [Miniconda](https://docs.conda.io/en/latest/miniconda.html), but with:
```

I'm not sure iv'e seen conda-forge capitalized ever.

Reply:
IMO it should be, but I'm not about to press that issue here :P